content/04.use-cases-b.md

```diff
@@ -20,16 +20,17 @@ If you wish to use your own domain name, you can purchase a domain for your GitH
-GitHub makes it easy to store and share a variety of data files in the cloud. 
-But for a variety of reasons (e.g., privately owned company, ability to make repositories private, accounts can be deleted at will) GitHub is not considered a long-term data or code repository like [zenodo](https://zenodo.org/) and [figshare](https://figshare.com/) [@doi:10.1038/538127a] [@doi:10.1371/journal.pcbi.1004947.]. 
+*Contributors to this section: Rob Crystal-Ornelas, Emma Hudgins, Dylan Gomes*  
+GitHub makes it easy to store and share a variety of data files in the cloud. If a repository is made "public" the URL to the repository can be shared freely with others.
+However, for a variety of reasons (e.g., privately owned company, ability to make repositories private, accounts can be deleted at will) GitHub is not considered a long-term data or code repository like [zenodo](https://zenodo.org/) and [figshare](https://figshare.com/) [@doi:10.1038/538127a] [@doi:10.1371/journal.pcbi.1004947.]. 
 Also, unlike the long-term repositories, GitHub does not issue Digital Object Identifiers (DOIs) for content uploaded to their servers. 
 DOIs are persistent and unique alpha-numeric IDs assigned to research products like papers, code, and data. 
 DOIs allows tracking and citing research products. 
 For this reason, scientists who share code and data through GitHub are strongly encouraged to also submit GitHub repository content to a long-term data archive [@doi:10.1029/2021EA001797]. Fortunately, both long-term repositories mentioned above (zenodo and figshare) have integrations with GitHub which facilitates archiving a snapshot of all repository content with the click of a button.  
 
-GitHub Linking with Zenodo, etc. to achieve a DOI helps work become findable, gives proper attribution, and that can ensure long-term stability (Hampton et al. 2015)
-Another key step researchers should consider taking when they receive a DOI for the content they archive on GitHub is choose a usage license [[@10.1016/j.patter.2021.100206]. This helps
+Linking one's GitHub repository with Zenodo, etc. to achieve a DOI helps work become findable, gives proper attribution, and that can ensure long-term stability (Hampton et al. 2015). Thus, when researchers wish to include data and code with their publications, they aught to reference a DOI from a long-term storage site, rather than a URL from GitHub (which can change or be deleted). Additionally, referencing a DOI for data and code is preferable to submitting these as supplementary materials to the journal, as supplementary materials are more difficult to find and reuse (i.e. often not centralized and searchable in a database) and not necessarily permanent (as most journals offer no guarantee of long-term storage).
+
+Another key step researchers should consider taking when they receive a DOI for the content they archive on GitHub is choose a usage license [[@10.1016/j.patter.2021.100206]. This helps...
```

Comment:
This could also be a good place to mention that GitHub will put a "cite this repository" link in the About section on the right side of the page if it finds an appropriate CITATION file in the repo (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files).

Reply:
I think we are arguing that GitHub is NOT a good long-term storage option, so we probably shouldn't suggest that people cite GitHub repositories, right? Or is this a way to link to a Dryad/Zenodo/etc. DOI reference? Or am I missing something?